akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpMessageParser.scala

```diff
@@ -46,6 +56,13 @@ private[http] abstract class HttpMessageParser[Output >: MessageOutput <: Parser
         if (self.onUpstreamFinish()) ctx.finish() else ctx.absorbTermination()
     }
 
+  final def onPush(input: SessionBytes): Output = {
```

Comment:
This didn't override anything but I changed the names so it's clearer which method calls which.